src/components/Viewer/Painting/Painting.tsx

```diff
@@ -36,11 +37,26 @@ const Painting: React.FC<PaintingProps> = ({
 
   const handleToggle = () => setIsInteractive(!isInteractive);
 
-  const handleCoiceChange = (value) => {
+  const handleChoiceChange = (value) => {
     const index = painting.findIndex((resource) => resource.id === value);
     setAnnotationIndex(index);
   };
 
+  const customDisplay = customDisplays.find((display) => {
+    let match = false;
+    const { canvasId, paintingFormat } = display.target;
+    if (canvasId) {
+      match = canvasId.includes(activeCanvas);
+    }
+    if (paintingFormat) {
+      match = paintingFormat === painting[annotationIndex]?.format;
```

Comment:
Going with the array notion above, this would need to be some form of `includes()`